SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKSimulatedPlayerDeployment.h

```diff
@@ -63,6 +63,9 @@ class SSpatialGDKSimulatedPlayerDeployment : public SCompoundWidget
 	/** Delegate called when the user has picked a path for the primary launch configuration file */
 	void OnPrimaryLaunchConfigPathPicked(const FString& PickedPath);
 
+	/** Delegate to commit deployment tags */
+	void OnDeploymentTagsCommited(const FText& InText, ETextCommit::Type InCommitType);
```

Comment:
```suggestion
	void OnDeploymentTagsCommitted(const FText& InText, ETextCommit::Type InCommitType);
```

Reply:
Wait, these are all misnamed...
this is consistent with existing naming..
if we want to do it, do it as a separate task and fix them all.

Reply:
I think it's best to fix this instance in this PR and then create a ticket to fix up all the other ones as well